README.md

```diff
@@ -6,17 +6,26 @@ Repository for Lukim Gather Mobile App
 - [Requirement](#requirement)
 - [Install](#install)
 - [Translation](#translation)
+- [Testing](#testing)
 - [License](#license)
 
 ## Requirement
 Lukim Gather apps targets iOS 11.0 and Android 5.0 (API 21) or newer.
 
 ## Install
-You can download and install the latest release from [Google Play Store](https://play.google.com/store/apps/details?id=org.lukimgather).
+Lukim Gather apps is now available on the Google Play Store and App Store
+<br/>
+<a href="https://play.google.com/store/apps/details?id=org.lukimgather">
+  <img alt="Get it on Google Play" height="80" src="https://play.google.com/intl/en_us/badges/images/generic/en_badge_web_generic.png" />
+</a>
+<br/>
+<a href="https://apps.apple.com/np/app/lukim-gather/id6443612199">
+  <img alt="Download on App Store" height="56" src="https://developer.apple.com/news/images/download-on-the-app-store-badge.png" />
+</a>
 
 ## Translation
 
-To translate Lukim Gather visit [Hosted Weblate](https://hosted.weblate.org/projects/lukim-gather/mobile-app/)
+To translate Lukim Gather visit [Hosted Weblate]()
```

Comment:
@kamal09rumba are we moving the actual link to the website?

Reply:
removed mistakenly